projects/ad74413r/src.mk

```diff
@@ -1,16 +1,3 @@
-include $(PROJECT)/src/platform/$(PLATFORM)/platform_src.mk
-include $(PROJECT)/src/examples/examples_src.mk
-
-SRCS += $(PROJECT)/src/platform/$(PLATFORM)/main.c
```

Comment:
all the files/subfolders in the project folder are included automatically if they follow the file tree pattern, so we can remove their explicit inclusion